src/platform/windows/mod.rs

```diff
@@ -0,0 +1,1486 @@
+// Copyright 2016 The Servo Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use bincode::serde::DeserializeError;
+use bincode;
+use kernel32;
+use libc::intptr_t;
+use serde;
+use std::cell::{Cell, RefCell};
+use std::cmp::PartialEq;
+use std::default::Default;
+use std::env;
+use std::ffi::CString;
+use std::io::{Error, ErrorKind};
+use std::marker::{Send, Sync, PhantomData};
+use std::mem;
+use std::ops::{Deref, DerefMut};
+use std::ptr;
+use std::slice;
+use uuid::Uuid;
+use winapi::{HANDLE, INVALID_HANDLE_VALUE, LPVOID};
+use winapi;
+use super::Incrementor;
+
+lazy_static! {
+    static ref CURRENT_PROCESS_ID: winapi::ULONG = unsafe { kernel32::GetCurrentProcessId() };
+    static ref CURRENT_PROCESS_HANDLE: intptr_t = unsafe { kernel32::GetCurrentProcess() as intptr_t };
```

Comment:
@retep998 actually we discussed this before (GitHub is just poor at recognising comments that still apply to newer code revisions...): indeed raw pointers can't be used directly in any kind of `static` (not just the "lazy" variety), as they are not thread-safe -- you would have to protect it with a mutex, which was my first idea.

However, since it turns out the Windows `HANDLE` is not actually a *real* pointer (even if formally it is declared as one in the API), just casting it as an `int` type is indeed the right thing to do -- and in fact ideally this probably should be done for *all* occurrences of `HANDLE`...

(However, @vvuk is already fed up with the delays in this PR as it is; and thus such non-essential improvements will have to wait for follow-up PRs :-) )